app/services/idv/lexisnexis_instant_verify.rb

```diff
@@ -0,0 +1,34 @@
+module Idv
+  class LexisnexisInstantVerify
```

Comment:
I missed it in the original PR but we are very consistent at casing this `LexisNexis` in all our other files (`lexis_nexis` in the filename), not blocking this PR but can we get another PR to clean that up?

Reply:
I wish we were more consistent between our class names and our config names, since I've been tripped up by this before. I'll make a PR to fix.

Reply:
Yeah configs are inconsistent which is a bummer, but this module name style and the filenames are thankfully very consistent